include/logger/logger.hpp

```diff
@@ -32,14 +32,16 @@ enum LogLevel {
 	SILENT = 0,
 	ERROR = 1,
 	INFO = 2,
-	DEBUG = 3
+	DEBUG = 3,
+	VERBATIM = 4
 };
 
-const char LogLevelStrings[][7] ={
+const char LogLevelStrings[][9] ={
```

Comment:
Why 9 instead of 8?

Reply:
I added a new VERBATIM entry which is 9 characters long. But I don't actually use it, so I can remove it and change it back to 7.